File: app/src/main/java/com/paycraft/ticketbooker/models/Stations.java
package com.paycraft.ticketbooker.models;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

import com.google.gson.annotations.SerializedName;

@Entity
public class Stations {

    @PrimaryKey
    @ColumnInfo(name = "id")
    @SerializedName("id")
    public int mStationId;

    @ColumnInfo(name = "source")
    @SerializedName("source")
    public String mSource;

    @ColumnInfo(name = "destination")
    @SerializedName("destination")
    public String mDestination;

    @ColumnInfo(name = "price")
    @SerializedName("price")
    public String mPrice;

    public Stations(int mStationId, String mSource, String mDestination, String mPrice) {
        this.mStationId = mStationId;
        this.mSource = mSource;
        this.mDestination = mDestination;
        this.mPrice = mPrice;
    }

    public int getmStationId() {
        return mStationId;
    }

    public void setmStationId(int mStationId) {
        this.mStationId = mStationId;
    }

    public String getmSource() {
        return mSource;
    }

    public void setmSource(String mSource) {
        this.mSource = mSource;
    }

    public String getmDestination() {
        return mDestination;
    }

    public void setmDestination(String mDestination) {
        this.mDestination = mDestination;
    }

    public String getmPrice() {
        return mPrice;
    }

    public void setmPrice(String mPrice) {
        this.mPrice = mPrice;
    }

    @Override
    public String toString() {
        return "Stations{" +
                "mStationId=" + mStationId +
                ", mSource='" + mSource + '\'' +
                ", mDestination='" + mDestination + '\'' +
                ", mPrice='" + mPrice + '\'' +
                '}';
    }
}

File: app/src/main/java/com/paycraft/ticketbooker/models/Stations.kt
package com.paycraft.ticketbooker.models

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

import com.google.gson.annotations.SerializedName

@Entity
class Stations(@field:PrimaryKey
               @field:ColumnInfo(name = "id")
               @field:SerializedName("id")
               var mStationId: Int, @field:ColumnInfo(name = "source")
               @field:SerializedName("source")
               var mSource: String, @field:ColumnInfo(name = "destination")
               @field:SerializedName("destination")
               var mDestination: String, @field:ColumnInfo(name = "price")
               @field:SerializedName("price")
               var mPrice: String) {

    fun getmStationId(): Int {
        return mStationId
    }

    fun setmStationId(mStationId: Int) {
        this.mStationId = mStationId
    }

    fun getmSource(): String {
        return mSource
    }

    fun setmSource(mSource: String) {
        this.mSource = mSource
    }

    fun getmDestination(): String {
        return mDestination
    }

    fun setmDestination(mDestination: String) {
        this.mDestination = mDestination
    }

    fun getmPrice(): String {
        return mPrice
    }

    fun setmPrice(mPrice: String) {
        this.mPrice = mPrice
    }

    override fun toString(): String {
        return "Stations{" +
                "mStationId=" + mStationId +
                ", mSource='" + mSource + '\''.toString() +
                ", mDestination='" + mDestination + '\''.toString() +
                ", mPrice='" + mPrice + '\''.toString() +
                '}'.toString()
    }
}
